ivy/functional/frontends/tensorflow/general_functions.py

```diff
@@ -343,3 +343,12 @@ def where(condition: ivy.array, x=None, y=None, name=None):
 
 def roll(input, shift, axis, name=None):
     return ivy.roll(input, shift, axis=axis)
+
+
+@to_ivy_arrays_and_back
+def repeat(
+        input,
+        repeats,
+        axis = None
```

Comment:
the `name` parameter is missing here, the function signature should match the one on TensorFlow framework.

Reply:
Hi @CatB1t, thanks for spotting this.  It should be fixed mow.